app/logic/apiEndpoint.py

```diff
@@ -0,0 +1,48 @@
+from flask import jsonify
+from datetime import datetime
+from app.config.loadConfig import*
+
+from app.models.laborStatusForm import LaborStatusForm
+from app.models.department import Department
+from app.models.formHistory import FormHistory
+from app.models.term import Term
+
+def getLaborInformation(orgCode = "", bNumber = ""):

```

Comment:
Consider making the function definition look like this instead:
`def getLaborInformation(*, orgCode = "", bNumber = ""):`
Doing this makes it so we can't pass in these parameters using positional arguments and given that these pieces of information aren't particularly obvious, it's probably best we enforce use of keyword arguments here.